### ISSUE TITLE
Evaluation on held out training data

### ISSUE THREAD

**Issue by ananyahjha93** [google-research/bert#272]
Does the run_pretraining.py script show its final evaluation result on held out training data? There is an 'is_training' param in the 'input_fn_builder' function but that just seems to control shuffling and parallel reading of data during training. The same 'input_files' argument is passed to both the training and validation parts of the function so I am not sure exactly where does the script keep a separate held out training set for evaluation of the trained model.

**Comment by pretidav** [comment]
> Does the run_pretraining.py script show its final evaluation result on held out training data? There is an 'is_training' param in the 'input_fn_builder' function but that just seems to control shuffling and parallel reading of data during training. The same 'input_files' argument is passed to both the training and validation parts of the function so I am not sure exactly where does the script keep a separate held out training set for evaluation of the trained model.

same question, have someone found this out?
